src/Core/Silk.NET.Core/Native/ComObject.cs

```diff
@@ -0,0 +1,47 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Silk.NET.Core.Native
+{
+    public unsafe class ComObject : IDisposable
```

Comment:
explain to me why this has to be in Core please?

Reply:
This preps Silk.NET.Core for any kind of COM interop. There has never been any guidelines for what constitutes a core addition, just whatever is common across some assemblies.